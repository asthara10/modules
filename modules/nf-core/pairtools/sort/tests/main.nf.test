nextflow_process {

    name "Test Process PAIRTOOLS_SORT"
    script "../main.nf"
    process "PAIRTOOLS_SORT"

    tag "modules"
    tag "modules_nfcore"
    tag "pairtools"
    tag "pairtools/sort"

    test("test_pairtools_sort") {


        when {
            process {
                """
                
                input[0] = [ [ id:'test', single_end:false ], // meta map
              file(params.test_data['generic']['pairtools']['mock_pairsam'], checkIfExists: true) ]

                """
            }
        }

        then {
            assertAll(
                { assert process.success },
                { assert snapshot(process.out.versions).match("versions") }
                { assert snapshot(process.out).match() }
            )
        }

    }

}
