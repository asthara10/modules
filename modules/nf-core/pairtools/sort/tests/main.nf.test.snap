{
    "versions": {
        "content": [
            [
                
            ]
        ],
        "timestamp": "2023-11-24T13:46:06.850892"
    }
}